src/plots/piePlot.ts

```diff
@@ -157,6 +157,42 @@ export module Plots {
       return this;
     }
 
+    /**
+     * Returns the PlotEntity at the query point.
+     * 
+     * @param {Point} queryPoint
+     * @param {PlotEntity} The PlotEntity at the point, or null if no PlotEntity can be found. 
+     */
+    public entityAt(queryPoint: Point): PlotEntity {
+      var bbox = d3.select(this._boundingBox[0][0]);
+      var center = { x: +bbox.attr("width") / 2, y: +bbox.attr("height") / 2 };
```

Comment:
Just use `this.height()/2` and `this.width()/2`.

We should probably have a private `_center()` method.